aspnet/getting-started/installing-on-mac.rst

```diff
@@ -26,13 +26,19 @@ You can also install ASP.NET 5 from the command-line. There are a few steps invo
 Install the .NET Version Manager (DNVM)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Use the .NET Version Manager (DNVM) to install different versions of the .NET Execution Environment (DNX) on OS X.
+To install DNVM:
 
-To install DNVM run the following::
+#. Open a Terminal.
+#. Type ``cd ~/`` to go to your home folder.
+#. Enter ``touch .bash_profile`` to create a new bash profile.
+#. Run the following ``curl`` command::
 
-  curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=dev sh && source ~/.dnx/dnvm/dnvm.sh
+	curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=dev sh && source ~/.dnx/dnvm/dnvm.sh
 
-Once this step is complete you should be able to run ``dnvm`` and see some help text.
+#. Run ``dnvm list`` to show the default DNX version
```

Comment:
Generally at this point you haven't installed any DNXs yet (installing one is the next step), so you won't have a default DNX version.